corehq/apps/hqadmin/tasks.py

```diff
@@ -48,3 +58,58 @@ def check_non_dimagi_superusers():
     if non_dimagis_superuser:
         _soft_assert_superusers(
             False, "{non_dimagis} have superuser privileges".format(non_dimagis=non_dimagis_superuser))
+
+
+@task(queue="email_queue", bind=True, acks_late=True)
```

Comment:
I don't think you need to do `bind=True` here since you aren't using the task instance anywhere. 
Also, why `acks_late`? I think that would end up sending people multiple emails if the task fails.

Reply:
Copied from somewhere else. Will remove, although I don't think multiple emails are an issue because this doesn't retry if it fails (as opposed to `send_html_email_async`).

Reply:
https://github.com/dimagi/commcare-hq/pull/20416/commits/b972a8f5692cda5a7a6ff3676ebb2424904f3931